tests/test_model.py

```diff
@@ -42,6 +42,7 @@
     can_finetune,
     create_package_rasa,
     get_latest_model,
+    get_local_model,
```

Comment:
minor thing: Could we please also import the things by their module here instead of importing the functions directly?